<launch>
    <node pkg="multi_robot_comunication" exec= "robot_1" name= "robot_1" output= "screen"/>
    <node pkg="multi_robot_comunication" exec= "robot_2" name= "robot_2" output= "screen"/>
    <node pkg="multi_robot_comunication" exec= "robot_3" name= "robot_3" output= "screen"/>
</launch>

